src/style/custom.css

```diff
@@ -1,3 +1,53 @@
+@responsive{
+  .bg-cover-image{
+         background-image: url('../static/svg/yellowfooterbig.svg');
+         background-size: 100% 100%;
+         height: 473px;
+     } 
+     .lg\:bg-cover-image {
+       background-image: url('../static/svg/yellowfooterbig.svg');
+       background-size: auto 100%;
+       height: 470px;
+     }
+ 
+     .bg-oval{
+       background-image: url('../static/svg/ovals.svg');
+       position: absolute;
+       mix-blend-mode:multiply;
+       padding-top: 50px;
+       background-repeat: no-repeat;
+       background-size: cover;
+       width: 293px;
+       height: 409px;
+       right: 0;
+       top: 30px;
+    } 
+     .lg\:bg-oval {
+       background-image: url('../static/svg/ovals.svg');
+       width: 211px;
+       height: 241px;
+       top: 20px;
+    } 
+    .social-icons{
+      position: absolute;
+      right: 0;
+      z-index: 1;
+      padding-right: 64px;
+      padding-bottom: 128px;
+    }
+    .lg\:social-icons{
+      padding-right: 40px;
+      padding-bottom: 230px;
+    }
+    .overflow-ovals{
```

Comment:
if we use it only on one place don't put it here and create styled component instead

Reply:
done